WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -58,6 +61,9 @@
     private boolean mEmailFollowersFetchRequestInProgress;
     private boolean mCanRefreshEmailFollowers;
     private int mEmailFollowersLastFetchedPage;
+    private boolean mViewersEndOfListReached;
```

Comment:
I think end of list is pretty clear, I've added a comment nevertheless.

P.S: Tracking a list view's end of list reached on an **Activity** seems pretty far fetched to me.